apps/fabric-website/src/styles/_mixins.scss

```diff
@@ -38,9 +40,9 @@
   position: relative;
 
   &::after {
-    bottom: 0;
+    bottom: -1px;
```

Comment:
Same here, i'm not sure why we need to have a negative value. I would try and make a variable for this or reshape the containing elements so everything fits perfectly.